### PR TITLE
feat:added wishlist component

### PR DESCRIPTION
**Pull request by JantuDeb**
### Wishlist

1. Add to cart.
2. Remove from wishlist.
3.  modify the quantity
image

### CODE REVIEW

**Review by Omkar-Chaskar — APPROVED** [review]
Good to merge

File: styles.css [hunk 1]
```diff
@@ -89,6 +89,10 @@ footer{
   .container{
     margin-top: 4rem;
   }
+  .wish-list-product{
+    width: 100%;
+    max-width: 50%;
+  }
```
Comment (rkapoor10) [inline styles.css:95]: you have used wish-list-product width 100% 2 times

File: styles.css [hunk 2]
```diff
@@ -97,4 +101,8 @@ footer{
 
 .border-bottom-dotted{
   border-bottom: 1px dotted var(--fu-gray-500);
+}
+
+.wish-list-product{
+  width: 100%;
```
Comment (rkapoor10) [inline styles.css:107]: here

File: pages/html/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+        </div>
+    </main>
+
+    <!-- FOOTER -->
+    <footer class="flex wrap py-4 bg-red p-2 space-evenly text-primary-dark">
+        <div>
+            <h3>About Us</h3>
+            <ul class="list-unstyled">
+                <div>
+                    <li>Our Stury</li>
```
Comment (rkapoor10) [inline pages/html/wishlist.html:163]: check the spelling of story

File: pages/html/wishlist.html [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+                    <li>FAQs</li>
+                </a>
+            </ul>
+        </div>
+        <div>
+            <h3>Stay Up To Date</h3>
+            <ul class="list-unstyled flex justify-between">
+                <a href="#"><i class="fab fa-youtube fa-2x"></i></a>
+                <a href="#"><i class="fab fa-twitter fa-2x"></i></i></a>
+                <a href="#"><i class="fab fa-github fa-2x"></i></a>
```
Comment (rkapoor10) [inline pages/html/wishlist.html:217]: add your github link

**Review by rkapoor10 — APPROVED** [review]
The code looks good to merge

**Pull request merged**